spec/spec.md

```diff
@@ -1223,16 +1243,12 @@ credentials within the target data structure.
     },
     {
       "@context": "https://www.w3.org/2018/credentials/v1",
-      "id": "https://eu.com/claims/DriversLicense",
-      "type": ["EUDriversLicense"],
```

Comment:
From vc-data-model:
> The value of the type property MUST be, or map to (through interpretation of the @context property), one or more URIs.

You've removed the Type. I think this makes it no longer a VC. Is that the intent?

Reply:
Wasn't the intent, so I guess I need to make up some random string to stick in type?

Reply:
+1

Reply:
according to the VC spec, the first URI in the type field MUST be `VerifiableCredential` anyway

Reply:
cc @csuwildcat i think we're good to go after this